packages/java-edition/test/mcfunction/parser/argument.spec.ts

```diff
@@ -321,7 +342,7 @@ describe('mcfunction argument parser', () => {
 								import.meta.url,
 							),
 							value: testParser(argument(treeNode)!, string, {
-								project: { meta },
+								project,
```

Comment:
this change is the main "bug fix" that allows us to actually specify a version in this tests now